### PR TITLE
MSTR-120: Render markdown back to HTML

### PR DESCRIPTION
**Pull request by masmerino13**

### CODE REVIEW

File: docs/monster-ui/markdownToHtml().md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+title: markdownToHtml()
+
+# monster.ui.markdownToHtml()
+The `monster.ui.markdownToHtml` render markdown content back to HTML.
```
Comment (guillegr123) [inline docs/monster-ui/markdownToHtml().md]:
```suggestion
The `monster.ui.markdownToHtml` renders markdown content as HTML.
```

File: docs/monster-ui/markdownToHtml().md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+## Syntax
+```javascript
+monster.ui.markdownToHtml(content);
+```
+
+### Parameters
+Key | Description | Type | Default | Required
+:-: | --- | :-: | :-: | :-:
+`content` | Markdown content that needs to be rendered to HTML | `String` | | `true`
```
Comment (guillegr123) [inline docs/monster-ui/markdownToHtml().md]:
```suggestion
`content` | Markdown content to be rendered as HTML | `String` | | `true`
```

File: docs/monster-ui/markdownToHtml().md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+## Description
+This helper will allows yo to render markdown content to HTML
```
Comment (guillegr123) [inline docs/monster-ui/markdownToHtml().md]:
```suggestion
This helper allows you to render markdown content as HTML
```

File: docs/monster-ui/markdownToHtml().md [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+
+## Example
+```javascript
+var markdownContent = '**Hello world**',
+  description = monster.ui.markdownToHtml(markdownContent);
```
Comment (guillegr123) [inline docs/monster-ui/markdownToHtml().md]:
```suggestion
    description = monster.ui.markdownToHtml(markdownContent);
```

File: docs/monster-ui/markdownToHtml().md [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+
+  template.find('.description').html(description);
```
Comment (guillegr123) [inline docs/monster-ui/markdownToHtml().md]:
```suggestion
template.find('.description').html(description);
```

File: src/js/lib/monster.ui.js [hunk 6]
```diff
@@ -1488,6 +1489,14 @@ define(function(require) {
 			}, options));
 		},
 
+		/**
+		 * Render markdown back to HTML
```
Comment (guillegr123) [inline src/js/lib/monster.ui.js]:
```suggestion
		 * Render markdown as HTML
```

**Comment by masmerino13** [comment]
@guillegr123, yep, I saw some examples like that, the reason why I decide to use an external lib is because this is the most popular sugestion in the forums, when someone ask for this using `simplemde`.

BTW, by using your suggestion we don't need to import/load a new library in the core (which is good :))

Let do some test and will back to you with some updates.

Thanks!

**Comment by masmerino13** [comment]
Hi @guillegr123, after testing and looking the docs the lib editor (simplemde) needs a textarea element as required.

**Pull request merged**